packages/datatrak-web-server/src/routes/ProjectRoute.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate'; 
+import { WebServerProjectRequest } from '@tupaia/types';
+
+export type ProjectRequest = Request<
+  WebServerProjectRequest.Params,
+  WebServerProjectRequest.ResBody,
+  WebServerProjectRequest.ReqBody,
+  WebServerProjectRequest.ReqQuery
+>;
+
+export class ProjectRoute extends Route<ProjectRequest> {
+  public async buildResponse() {
+    const {
+      ctx,
+      params: { projectCode },
+    } = this.req;
+
+    const { projects } = await ctx.services.webConfig.fetchProjects();
```

Comment:
The plan is to phase out web-config-server because it's a bit of a frankenstein legacy server (which is why Ethan has been re-writing backend stuff so much on tupaia-web) so we should get the projects from central server instead.

Reply:
As I mentioned below, let's split out the work to refactor this into a new ticket.

Reply:
Sure - the reason for this is that getting it from the central server doesn't return things like `name`, `hasAccess` etc, which we need